microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -87,31 +95,54 @@ For full support for the latest APIs in a release version of the SDK, the Runtim
+| _Stable_ | 2. Then the API is promoted to Stable in the Prerelease SDK.  The API isn't in a Release SDK yet. |
+| _Release_ | 3. Then the API is promoted to be included in the Release SDK.  This typically happens 1 month after the API is promoted to Stable in a Prerelease SDK.  The API also remains in the Prerelease SDK.  |
+
+
+<!-- ------------------------------ -->
+#### Developing with Experimental APIs and providing feedback
+
+The Experimental APIs in a WebView2 Prerelease SDK package aren't guaranteed to be forward-compatible, and might be removed in future Runtime updates.
 
+For full support of Experimental APIs, use a Microsoft Edge preview channel, not the WebView2 Evergreen Runtime.  When a _prerelease_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  Soon after that, the Prerelease SDK also works with the Beta and Dev channels.
 
-### Developing with experimental APIs and providing feedback
+Use a Prerelease SDK to try out new, Experimental APIs early and provide feedback before the Experimental APIs are promoted to become Stable, forward-compatible APIs.
 
-The experimental APIs in a WebView2 SDK _prerelease_ package aren't guaranteed to be forward-compatible and might be removed in future Runtime updates.  When a _prerelease_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  Soon after that, the prerelease SDK also works with the Beta and Dev channels.  Use a prerelease SDK to try out new APIs early and provide feedback before the new APIs are promoted to become stable, forward-compatible APIs.
+*  The Experimental APIs (in a Prerelease SDK) aren't guaranteed to be forward-compatible.
+*  The Stable APIs (in a Prerelease SDK) are forward-compatible.
+*  The Release APIs (in a Release SDK) are also forward-compatible.
 
-For full support of experimental APIs, use a Microsoft Edge preview channel, not the WebView2 Evergreen Runtime.  Any experimental APIs that are in a prerelease SDK aren't guaranteed to be forward-compatible.  The APIs that are in an SDK _release_ version are forward-compatible.  For more information, see [Forward compatibility of APIs](#forward-compatibility-of-apis), above.
+For more information, see [Forward compatibility of APIs](#forward-compatibility-of-apis), above.
 
-The WebView2 team is seeking feedback on experimental WebView2 APIs that might be promoted to Stable in future releases.  The experimental APIs are indicated as "experimental" in the WebView2 SDK Reference documentation.
+The WebView2 team is seeking feedback on Experimental WebView2 APIs that might be promoted to Stable in future Prerelease SDKs (and then promoted to Release in Release SDKs).  The Experimental APIs are indicated as "experimental" in the WebView2 SDK Reference documentation.
+<!-- todo: "experimental" is missing from .NET API Ref:
```

Comment:
@peiche-jessica @novac42 -- There is an "Experimental" Alert in Win32 as claimed, but not in .NET or WinRT API Ref docs.

.NET:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2webmessagereceivedeventargs.additionalobjects?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true

Win32:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalwebmessagereceivedeventargs?view=webview2-1.0.1724-prerelease&preserve-view=true#get_additionalobjects

Reply:
I'm not sure if this particular one is an experimental API as it doesn't have 'experimental' in API naming like Win32's. Would need @peiche-jessica 's help to confirm.

Reply:
Here is evidence in RelNotes that that release version of that .NET API is Experimental just like same as Win32: 
https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#101724-prerelease

@novac42 -- only Win32 types have "Experimental" when experimental, not .NET types.

Reply:
Resolving conversation as out of scope for PR.  Described the 'experimental' Note that's in Win32.